roles/install_initial/tasks/main.yml

```diff
@@ -46,3 +46,6 @@
     npm:
       name: yuglify
       global: yes
+
+  - name: Install CodeDeploy agent
+    raw: cd /home/ubuntu && sudo wget https://aws-codedeploy-eu-west-2.s3.eu-west-2.amazonaws.com/latest/install && sudo chmod +x ./install && sudo ./install auto
```

Comment:
@symroe this works but I'm guessing there is probably a nicer way to do it? Any suggestions?

Reply:
First off, if we use ImageBuilder then we might want something like this, unless we decide to use Ansible to make the images.

For the time being, let's assume we will use Ansible. If you wanted to be more ansible like here, I expect you could use [`get_url`](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/get_url_module.html) to download the package and then `shell` with `become: true`:

```yml
- name: Download
  get_url:
    url: "https://aws-codedeploy-eu-west-2.s3.eu-west-2.amazonaws.com/latest/install"
    dest: /home/ubuntu/codedeploy
    mode: 0755

- name: Install
  become: true
  shell: /home/ubuntu/codedeploy auto
```